src/tools/miri/tests/fail/data_race/stack_pop_race.rs

```diff
@@ -22,5 +22,4 @@ fn race(local: i32) {
     thread::yield_now();
     // Deallocating the local (when `main` returns)
     // races with the read in the other thread.
-    // Make sure the error points at this function's end, not just the call site.
```

Comment:
uh 😆 obviously intended change, but should we try to do something here in the future?

Reply:
Yeah... well TBH I am no longer sure that that's really the behavior we want here.

Deallocating locals has to happen after the return value go copied, and when there's something wrong with copying the return value clearly we want that to point at the caller. We used to have some very special hacks here to delay the "UB during ret val copy" error so that if there's also UB during deallocation, that takes precedent... but is that really a good idea?